packages/api/tests/endpoints/granules/helpers.js

```diff
@@ -0,0 +1,122 @@
+const {
+  translateApiGranuleToPostgresGranule,
+  translateApiFiletoPostgresFile,
+} = require('@cumulus/db');
+
+const {
+  putJsonS3Object,
+  s3PutObject,
+} = require('@cumulus/aws-client/S3');
+
+const {
+  getBucketsConfigKey,
+  getDistributionBucketMapKey,
+} = require('@cumulus/common/stack');
+
+const {
+  fakeGranuleFactoryV2,
+} = require('../../../lib/testUtils');
+
+const getPostgresFilesInOrder = async (knex, newGranule, filePgModel, postgresGranuleCumulusId) => {
+  const pgFiles = await Promise.all(newGranule.files.map(async (file) => {
+    const [pgFile] = await filePgModel.search(knex, {
+      granule_cumulus_id: postgresGranuleCumulusId,
+      file_name: file.fileName,
+    });
+    return pgFile;
+  }));
+  return pgFiles;
+};
+
+const generateMoveGranuleTestFilesAndEntries = async (params) => {
+  const {
+    t,
+    bucket,
+    secondBucket,
+    granulePgModel,
+    filePgModel,
+    granuleModel,
+    granuleFileName,
+    createPostgresEntries = true,
+  } = params;
+  const newGranule = fakeGranuleFactoryV2({ collectionId: t.context.collectionId });
+  // TODO: File factory?
```

Comment:
In this instance the only thing that the factory really provides is a random bucket/filename, except we don't want those to be random.   

The upside is if any schema change requires another field in the future and it's added, I suppose this would get updated:  904a45f69